python/dolfinx/io.py

```diff
@@ -62,17 +62,20 @@ def read_mesh(self, ghost_mode=cpp.mesh.GhostMode.shared_facet, name="mesh", xpa
         return mesh
 
 
-# Map from Gmsh string to DOLFIN cell type and degree
-_gmsh_cells = dict(tetra=("tetrahedron", 1), tetra10=("tetrahedron", 2), tetra20=("tetrahedron", 3),
-                   hexahedron=("hexahedron", 1), hexahedron27=("hexahedron", 2),
-                   triangle=("triangle", 1), triangle6=("triangle", 2), triangle10=("triangle", 3),
-                   quad=("quadrilateral", 1), quad9=("quadrilateral", 2), quad16=("quadrilateral", 3),
-                   line=("interval", 1), line3=("interval", 2), line4=("interval", 3),
-                   vertex=("point", 1))
+# Map from Gmsh float to DOLFIN cell type and degree
+# http://gmsh.info//doc/texinfo/gmsh.html#MSH-file-format
+_gmsh_to_cells = {1: ("interval", 1), 2: ("triangle", 1),
+                  3: ("quadrilateral", 1), 4: ("tetrahedron", 1),
+                  5: ("hexahedron", 1), 8: ("interval", 2),
+                  9: ("triangle", 2), 10: ("quadrilateral", 2),
+                  11: ("tetrahedron", 2), 12: ("hexahedron", 2),
+                  15: ("point", 1), 21: ("triangle", 3),
```

Comment:
I would have expected "point" to have degree 0.

Reply:
I can change it. For this, I was just following the previous `_gmsh_cells`.